src/Bridge/Doctrine/EventListener/WriteListener.php

```diff
@@ -42,7 +42,7 @@ public function __construct(ManagerRegistry $managerRegistry)
     public function onKernelView(GetResponseForControllerResultEvent $event): void
     {
         $request = $event->getRequest();
-        if ($request->isMethodSafe(false)) {
+        if ($request->isMethodSafe()) {
```

Comment:
We have to change the version constraint for `symfony/http-foundation` to `^4.3 || ^5.0`, because:

1. With `^4.0 < 4.3`, this would result in a `BadMethodCallException`.
2. With `^3.4`, this would result in the wrong value being returned.

Reply:
I don’t think so, we don’t support OPTIONS and TRACE (we were already using this method without arguments before IIRC).

Reply:
> we don’t support OPTIONS and TRACE

It's exactly why we're excluding any safe method here, including OPTIONS and TRACE.